Userland/Services/WebDriver/Session.cpp

```diff
@@ -45,7 +46,14 @@ ErrorOr<NonnullRefPtr<Core::LocalServer>> Session::create_server(NonnullRefPtr<S
         }
 
         dbgln("WebDriver is connected to WebContent socket");
-        m_web_content_connection = maybe_connection.release_value();
+        auto web_content_connection = maybe_connection.release_value();
+
+        String handle_name = MUST(String::formatted("window-{}"sv, next_handle_id));
```

Comment:
`auto handle_name` here, an also should be `release_value_but_fixme_should_propagate_errors`